tests/pyincore/analyses/buildingdamage/test_buildingdamage_offline.py

```diff
@@ -0,0 +1,75 @@
+import os
+
+from pyincore import IncoreClient, FragilityCurveSet, MappingSet, Tornado, GeoUtil, Dataset, Mapping, FragilityService
```

Comment:
GeoUitl is not used either, probably can be removed too

Reply:
Good catch. Removed :-)